### DISCUSSION TITLE
Picker Background Color

### DISCUSSION THREAD

**Discussion by BinaryAssault** [dotnet/maui#7759] [Q&A]
How is the background color set on a picker when it is clicked? I'd like it to stay the same between light and dark themes but none of the color properties seem to have any effect on the background of a picker.

**Comment by Symbai** [comment]
https://github.com/dotnet/maui/issues/7542
https://github.com/dotnet/maui/issues/7800

You're not the only one and this isn't the only control where certain colors cannot be changed by developers.
